lib/favorite_news_screen.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:count_up_app/news_webview.dart';
+import 'package:flutter/material.dart';
+
+import 'file_helper.dart';
+import 'model/article.dart';
+
+class FavoriteNewsScreen extends StatefulWidget {
+  const FavoriteNewsScreen({super.key});
+
+  @override
+  State<FavoriteNewsScreen> createState() => _FavoriteNewsScreenState();
+}
+
+class _FavoriteNewsScreenState extends State<FavoriteNewsScreen> {
+  late Future<List<Article>> _favorites;
+
+  @override
+  void initState() {
+    super.initState();
+    _favorites = readFavorites();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('お気に入りニュース'),
+        actions: <Widget>[
+          IconButton(
+            icon: const Icon(Icons.delete_outlined),
+            onPressed: () async {
+              // お気に入りをすべて削除
+              final favorites = await _favorites;
+              favorites.clear();
+              await writeFavorites(favorites);
```

Comment:
```suggestion
              await writeFavorites([]);
```

やってること変わらないのでこちらでも良いのですが、`cleanFavorite()` というメソッドを作ったほうが良いかと思います。

Reply:
```
Future<void> cleanFavorite() async {
    await writeFavorites([]);
  }
```
cleanFavorite()メソッドを作成し、空のリストを渡して更新するように修正しました。